frontend/src/App.js

```diff
@@ -1,15 +1,30 @@
-
-import './App.css';
+import "./App.css";
+import { useEffect } from "react";
 
 function App() {
+  var websocket = null;
```

Comment:
I'd recommend we stick to `let` and `const` declarators. Also save the WebSocket in a react state